### ISSUE TITLE
Update Rooted to v1.1

### ISSUE THREAD

**Issue by rockymine** [OvercastNetwork/Issues#1818]
**Update**: *Rooted v1.1*
After a while I've decided to update Rooted as it is in the repository. The "while" is standing for 8 months, a long time. I entirely redesigned the map with a more natural theme, it turned out great. The layout didn't really change:

- Fixed a small issue in symmetry
- 2 blocks longer wool-room lanes
- New spawn-lane

Some other changed I've done (with matic0basle):

- New Spawnkit
- Different wool-room loot
- Improved bedrock-barrier
- Thicker Soil
- and more...

In general we tried to improve the gameplay, all in all there are less obstacles etc.

**Links**
- [Download](http://www.mediafire.com/download/6l2rhn6qgl6okdq/Rooted+v1.1.zip)
- [XML](https://gist.github.com/rockymine/05af1f21456119e2a53f)
- [Pictures](http://imgur.com/a/AtHMZ)
- [Old](https://oc.tc/forums/topics/54041cb167d5225b0e008387)

*Thanks for reading!*

**Comment by thestrangeboy1** [comment]
There's a problem with no block 36 under the wool lanes when it goes 1 block above the void. Add some block 36 under there and also remove those unpruned blocks from the map. In case you need to do XML changes after doing that, here's what I did to fix most of the stuff: https://gist.github.com/thestrangeboy1/78ddf951734893e7a4bd

Comment back when you updated the download

**Comment by rockymine** [comment]
@thestrangeboy1: Thanks for telling me! Just a question, what do you exactly mean with the unturned blocks? The map was pruned...

**Comment by thestrangeboy1** [comment]
There was a strip of blocks left at one end of that resembled the superflat terrain.

**Comment by rockymine** [comment]
@thestrangeboy1 I did what you wanted. Had some difficulties to see the grass line first, found and fixed it though. Furthermore the wool-lane shall be fixed now. Another change I did is listed here:
- Decreased amount of wood in defenders chest, seeing that it was a ton of wood. I replaced two lines of the wood with some leaves.

http://www.mediafire.com/download/rx27didhu63uxxv/Rooted+v1.1.1.zip

Tell me if I missed anything ;)
